packages/axes/src/inputType/MoveKeyInput.ts

```diff
@@ -199,17 +199,22 @@ export class MoveKeyInput implements InputType {
   }
 
   private _attachEvent(observer: InputTypeObserver) {
+    const element = this.element;
+    if (!element) {
+      throw new Error("Element to connect input does not exist.");
+    }
     this._observer = observer;
-    this.element.addEventListener("keydown", this._onKeydown, false);
-    this.element.addEventListener("keypress", this._onKeydown, false);
-    this.element.addEventListener("keyup", this._onKeyup, false);
+    element.addEventListener("keydown", this._onKeydown, false);
+    element.addEventListener("keypress", this._onKeydown, false);
+    element.addEventListener("keyup", this._onKeyup, false);
     this._enabled = true;
   }
 
   private _detachEvent() {
-    this.element.removeEventListener("keydown", this._onKeydown, false);
-    this.element.removeEventListener("keypress", this._onKeydown, false);
-    this.element.removeEventListener("keyup", this._onKeyup, false);
+    const element = this.element;
+    element?.removeEventListener("keydown", this._onKeydown, false);
+    element?.removeEventListener("keypress", this._onKeydown, false);
+    element?.removeEventListener("keyup", this._onKeyup, false);
```

Comment:
You'd rather do it like:
```js
const element = this.element;
if (element) {
  element.removeEventListener("keydown", this._onKeydown, false);
  element.removeEventListener("keypress", this._onKeydown, false);
  element.removeEventListener("keyup", this._onKeyup, false);
}
```

Because simply, the one you wrote turns like this on tsc compiler:
```js
var element = this.element;
element === null || element === void 0 ? void 0 : element.removeEventListener("keydown", this._onKeydown, false);
element === null || element === void 0 ? void 0 : element.removeEventListener("keypress", this._onKeydown, false);
element === null || element === void 0 ? void 0 : element.removeEventListener("keyup", this._onKeyup, false);
```